scipy/ndimage/_measurements.py

```diff
@@ -1053,7 +1053,7 @@ def minimum(input, labels=None, index=None):
 
     Returns
     -------
-    minimum : float or list of floats
+    output : a scalor or list of integer or float based on input type.
```

Comment:
```suggestion
    output : a scalar or list of integesr or floats based on input type.
```
And below

Reply:
Thanks for review!!. sorry let me confirm one thing, `integesr` is a valid term?

Reply:
Ha sorry a typo from me!